Makefile

```diff
@@ -129,7 +129,7 @@ check:
 	  grep -vE '(declaration of err shadows|^vet: cannot process directory \.git)'
 	@echo "golint"
 	@! golint $(PKG) | \
-	  grep -vE '(\.pb\.go|embedded\.go|_string\.go|LastInsertId|sql\.y)' \
+	  grep -vE '(\.pb\.go|embedded\.go|_string\.go|LastInsertId|sql/parser/(yaccpar|sql\.y):)' \
```

Comment:
It is the "file" containing the generated parser code and is annotated in `sql.go` with:

```
//line yaccpar:1
```